tests/test_httpurl.py

```diff
@@ -275,6 +276,20 @@ def test_httpurl_read(url_v6_txt):
     assert u.read(byte=True) == v6_txt_contents().encode()
 
 
+def test_httpurl_find_all_files_and_rmdir(url_test_path):
+    """Test two methods which are not supported (implemented).:
+        - find_all_files()
+        - rmdir()
+    """
+    prefix = os.path.join(url_test_path, 'test_httpurl_find_all_files_and_rmdir')
+
+    with pytest.raises(NotImplementedError):
+        HTTPURL(prefix).find_all_files()
+
+    with pytest.raises(NotImplementedError):
+        AutoURI(prefix).rmdir()
```

Comment:
This would be better as two tests. Also is the `AutoURI` here a typo? Should it be `HTTPUrl`? If not then it is is confusing to have an `AutoURI` test here.

Reply:
Fixed as you suggested. Yes it was typo. It worked since `AutoURI` auto-converts its `__class__` into a detected type.